CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# ✨ Contributing to LWS ✨
```

Comment:
Should be changed to "Contributing to setup-vlang-action" or something similar.

Reply:
Haha copy past mistake XD
Thanks